### ISSUE TITLE
Fix error w/ 2020.2 in shorty-idea-plugin

### ISSUE THREAD

**Issue by nazmulidris** [r3bl-org/shorty-idea-plugin#8]
I get the following
```
Plugin Error
Plugin "Shorty - URL Shortener" is incompatible (until build 201.SNAPSHOT < WS-202.6397.88).
```

With
```
WebStorm 2020.2
Build #WS-202.6397.88, built on July 24, 2020
Licensed to Nazmul Idris
Subscription is active until May 17, 2021
Runtime version: 11.0.7+10-b944.20 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 10.15.6
GC: ParNew, ConcurrentMarkSweep
Memory: 725M
Cores: 16
Registry: ide.settings.move.mouse.on.default.button=true
Non-Bundled Plugins: GrepConsole, PsiViewer, com.alayouni.ansiHighlight, com.andrewbrookins.wrap_to_column, com.intellij.properties, com.github.jk1.ytplugin, com.jetbrains.hackathon.indices.viewer, com.mnw.tabmover, monokai-pro, izhangzhihao.rainbow.brackets
```

**Comment by nazmulidris** [comment]
Changes to `build.gradle.kt` file to make it work:

```
// See https://github.com/JetBrains/gradle-intellij-plugin/
intellij {
    // Information on IJ versions https://www.jetbrains.org/intellij/sdk/docs/reference_guide/intellij_artifacts.html
    // You can use release build numbers or snapshot name for the version.
    // 1) IJ Release Repository w/ build numbers https://www.jetbrains.com/intellij-repository/releases/
    // 2) IJ Snapshots Repository w/ snapshot names https://www.jetbrains.com/intellij-repository/snapshots/
    version = "202-EAP-SNAPSHOT" // You can also use LATEST-EAP-SNAPSHOT here.

    // Declare a dependency on the markdown plugin to be able to access the
    // MarkdownRecursiveElementVisitor.kt file. More info:
    // https://www.jetbrains.org/intellij/sdk/docs/basics/plugin_structure/plugin_dependencies.html
    // https://plugins.jetbrains.com/plugin/7793-markdown/versions
    setPlugins("java", "org.intellij.plugins.markdown:202.6397.21")
}
```

**Comment by nazmulidris** [comment]
Fix is in c5563f3263252e0cf111b930fa40a44148afe874
